lmdeploy/pytorch/weight_loader/model_weight_loader.py

```diff
@@ -0,0 +1,228 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import json
+import os.path as osp
+from contextlib import ExitStack, contextmanager
+from typing import Dict
+
+import torch
+from transformers.modeling_utils import load_state_dict
+from transformers.utils import (SAFE_WEIGHTS_INDEX_NAME, SAFE_WEIGHTS_NAME,
+                                WEIGHTS_INDEX_NAME, WEIGHTS_NAME)
+
+from lmdeploy.utils import get_logger
+
+from .adapter_weight_loader import AdapterWeightLoader
+
+logger = get_logger('lmdeploy')
+
+
+def _get_weight_type(model_path: str, use_safetensors: bool = None):
```

Comment:
`use_safetensors` can be {True, False, None}. Why not `True` or `False`?

Reply:
Align with transformers

https://github.com/huggingface/transformers/blob/dcdda5324bcc7a750b5e40e11dd795442204ff27/src/transformers/modeling_utils.py#L2813